src/current/v24.1/certificate-based-authentication-using-the-x509-subject-field.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Certificate-based authentication using multiple values from the X.509 Subject field
+summary: Learn about how to authenticate users using additional features already present in your X.509 security certificates
+toc: true
+docs_area: manage
+keywords: authentication, ldap, X.509, X509, tls
+---
+
+{% include_cached new-in.html version="v24.1" %} For customers that need to use their established Certificate Authority (CA) infrastructure to manage SQL user authentication, CockroachDB supports mapping of SQL user [roles]({% link {{ page.version.version }}/security-reference/authorization.md %}#roles) to values in the Subject field of the [X.509 certificate](https://en.wikipedia.org/wiki/X.509) used for TLS authentication.
+
+This mapping (hereafter referred to as "Subject mapping") can be used to automate the synchronization of SQL user roles with specific certificate attributes, leading to improved scalability of access control mechanisms.
+
+Subject mapping is useful if:
+
+- You run your own [Certificate Authority (CA)]({% link {{ page.version.version }}/security-reference/transport-layer-security.md %}#certificates-signing-trust-and-authority) infrastructure.
+- You need to use your existing CA infrastructure to manage [SQL user authentication]({% link {{ page.version.version }}/create-user.md %}#user-authentication).
+- You need to use the same CA for multiple CockroachDB clusters.
+
+For instructions showing how to map SQL user roles to values in the Subject field of the X.509 certificate, see the [Example](#example).
+
+{% include enterprise-feature.md %}
+
+## Related functionality
+
+Prior to CockroachDB v24.1, it was only safe to use certificates generated by CAs that placed restrictions on the `CommonName` field (so that only authorized users could get a cert with `CommonName=root`, for example). That meant that mapping the certificate principals to IP addresses, DNS names, and SQL users required using [the `--cert-principal-map` flag]({% link {{ page.version.version }}/cockroach-start.md %}#flags-cert-principal-map), which was [first added in v20.1]({% link releases/v20.1.md %}#v20-1-0-beta-3-changes-to-operational-workflows). That approach still works, but is superseded by the functionality described on this page, which should be much easier to use.
```

Comment:
yes that's my understanding - you can leave your certs as is and do CREATE ROLE or ALTER ROLE

updated to add "... since you do not need to restart nodes when you add or remove users." (with a link to example)